rms_fe/src/clients/auth.ts

```diff
@@ -0,0 +1,16 @@
+import { LoginCredentials } from "../types/api/auth"
+import axios from "axios"
+
+axios.defaults.withCredentials = true
```

Comment:
This is present in the `setupAxios` function so this is unnecessary and can be removed

Reply:
fixed